spec/classes/datadog_agent_reports_spec.rb

```diff
@@ -46,7 +46,7 @@
         end
 
         it do
-          should contain_file('/etc/dd-agent/datadog.yaml')\
+          should contain_file('/etc/datadog-agent/datadog-reports.yaml')\
```

Comment:
So no matter the agent version we use `/etc/datadog-agent` for report ? Won't it be confusing for people update the puppet classes but not the agent ?

Reply:
Yeah, this sucks a bit... I guess maybe `/etc/datadog/` is better. What I don't want to do is keep dd-agent around, because that will eventually be even more confusing for customers moving away from agent5.

Reply:
Indeed. I guess `/etc/datadog-agent` is the best choice then.